compiler/luci/pass/src/CircleOptimizer.cpp

```diff
@@ -548,49 +304,162 @@ void CircleOptimizer::optimize(loco::Graph *g) const
   {
     phase.emplace_back(std::make_unique<luci::SubstituteTransposeToReshapePass>());
   }
-  if (_options->query(Options::Algorithm::TransformMinMaxToRelu6Pass))
-  {
-    phase.emplace_back(std::make_unique<luci::TransformMinMaxToRelu6Pass>());
-  }
-  if (_options->query(Options::Algorithm::TransformMinReluToRelu6Pass))
-  {
-    phase.emplace_back(std::make_unique<luci::TransformMinReluToRelu6Pass>());
-  }
-  if (_options->query(Options::Algorithm::TransformSqrtDivToRsqrtMul))
-  {
-    phase.emplace_back(std::make_unique<luci::TransformSqrtDivToRsqrtMulPass>());
-  }
-  if (_options->query(Options::Algorithm::DecomposeHardSwishPass))
-  {
-    phase.emplace_back(std::make_unique<luci::DecomposeHardSwishPass>());
-  }
-  if (_options->query(Options::Algorithm::DecomposeSoftmaxPass))
+  if (_options->query(Options::Algorithm::RemoveRedundantReshape))
   {
-    phase.emplace_back(std::make_unique<luci::DecomposeSoftmaxPass>());
+    phase.emplace_back(std::make_unique<luci::RemoveRedundantReshapePass>());
   }
-  if (_options->query(Options::Algorithm::UnrollUnidirSeqLSTM))
+  if (_options->query(Options::Algorithm::RemoveRedundantTranspose))
   {
-    phase.emplace_back(std::make_unique<luci::UnrollUnidirectionalSequenceLSTMPass>());
+    phase.emplace_back(std::make_unique<luci::RemoveRedundantTransposePass>());
   }
 
+  // clang-format off
+  std::map<Options::Algorithm, std::unique_ptr<logo::Pass> (*)(void)> option_to_pass;
+
+  option_to_pass[Options::Algorithm::CommonSubExpressionElimination] =
+&createPassInstance<luci::CommonSubExpressionEliminationPass>;
```

Comment:
Ah, if you accepted to to use `// clang-format off`, the proposal was to put to single line
for better readability like;
```cxx
// before
  option_to_pass[Options::Algorithm::CommonSubExpressionElimination] =
&createPassInstance<luci::CommonSubExpressionEliminationPass>;

// after
  option_to_pass[Options::Algorithm::CommonSubExpressionElimination] = &createPassInstance<luci::CommonSubExpressionEliminationPass>;
```

Reply:
I got it:)